test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +37,40 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsAreNotNull() throws Exception {
```

Comment:
The test itself is correctly written test. 
But, is the method you are testing `elementsAreNotNull`? 
Remember the 3 part name format for JUnit test; the first part of the test name is the method you are testing.